crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -184,3 +194,59 @@ async fn get(db_conn: DbConnection) -> superposition::Result<Json<Vec<DefaultCon
     let result: Vec<DefaultConfig> = default_configs.get_results(&mut conn)?;
     Ok(Json(result))
 }
+
+pub fn get_key_usage_context_ids(
+    key: &str,
+    conn: &mut PooledConnection<ConnectionManager<PgConnection>>,
+) -> superposition::Result<Vec<String>> {
+    let result: Vec<Context> = contexts.load(conn).map_err(|err| {
+        log::error!("failed to fetch contexts with error: {}", err);
+        db_error!(err)
```

Comment:
Nor the DB error, user doesn't have to know that we have failed to load the context

Reply:
We can return the exact error from here.
But in the handler, its not really required to return the same error, why the user has to know that a db load failed to happen for context, they are not even making a request for context, its related to default_config, why to even provide a error that, its just an internal operation.

Reply:
yes agree on this